website/src/displaySecret/Secret.tsx

```diff
@@ -38,6 +40,16 @@ const RenderSecret = ({ secret }: { readonly secret: string }) => {
       >
         {secret}
       </Typography>
+	  <Button
+	  	onClick={() => setShowQr(!showQr)}
+	  >{(showQr ? t('secret.hideQrCode') : t('secret.showQrCode'))}</Button>
+      <Box sx={{ display:(showQr ? "flex" : "none"), justifyContent:"center", alignItems:"center", margin:5 }}>
+        <QRCode
+          size={512}
```

Comment:
Lets make the QR code a lot smaller, 150 looks alright.
```suggestion
          size={150}
```